### PR TITLE
compression with tinyPNG

### PR DESCRIPTION
**Pull request by TakshPanchal**
- Closes https://github.com/appwrite/appwrite/issues/4118

image

### CODE REVIEW

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

File: dart/compress_image/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    }
+  } catch (e) {
+    print("Failure in tinpng $e");
+    markFailure(res, e.toString());
+    return;
+  }
+}
+
+Future<void> _krakenio(
+    final res, final Uint8List binaryImage, final String token) async {}
```
Comment (stnguyen90) [inline dart/compress_image/lib/main.dart:34]: Home come this isn't implemented?

**Review by stnguyen90 — COMMENTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: dart/compress_image/lib/main.dart (hunk 1 above)
Comment (TakshPanchal) [reply dart/compress_image/lib/main.dart:34]: I got problem in kraken API

**Review by Meldiron — APPROVED** [review]

**Comment by Meldiron** [comment]
Thanks for the amazing PR 😇 I have added small changes.
Merging the PR now 🥳

**Pull request merged**